language/stdlib/modules/SlidingNonce.move

```diff
@@ -31,20 +37,20 @@ module SlidingNonce {
     /// Tries to record this nonce in the account.
     /// Returns 0 if a nonce was recorded and non-0 otherwise
     /// Reasons for nonce to be rejected:
-    /// * code 10001: This nonce is too old and impossible to ensure whether it's duplicated or not
-    /// * code 10002: This nonce is too far in the future - this is not allowed to protect against nonce exhaustion
-    /// * code 10003: This nonce was already recorded previously
+    /// * NONCE_TOO_OLD: This nonce is too old and impossible to ensure whether it's duplicated or not
+    /// * NONCE_TOO_NEW: This nonce is too far in the future - this is not allowed to protect against nonce exhaustion
+    /// * NONCE_ALREADY_RECORDED: This nonce was already recorded previously
```

Comment:
I know this is for documentation but it feels weird to describe the Errors where they are used, and not where they are defined.

Reply:
Good point :) I will move up to the point of definition.